server/accounts.go

```diff
@@ -2693,7 +2693,7 @@ func (a *Account) hasIssuer(issuer string) bool {
 // hasIssuerNoLock is the unlocked version of hasIssuer
 func (a *Account) hasIssuerNoLock(issuer string) bool {
 	// same issuer
```

Comment:
change the name of the function to match its functionality?
account IsIssuing?

Reply:
that name is correct - it is checking the issuer of the user JWT. The account has such an issuer key.

Reply:
ok